### DISCUSSION TITLE
Socket.io-parser doesn't decode for large size message

### DISCUSSION THREAD

**Discussion by llssyy1991** [socketio/socket.io#3750] [Q&A] [answered]
Hi, I got a message that over 150K from server side, and I have enable the debug mode inside console. I see the engine.io-client:socket has received the message, but socket.io-parser didn't parse it and so socket.io-client doesn't do the following work. Is there anyone got what's happening?

Thanks.

**Comment by darrachequesne** [comment, marked as answer]
We will need **A LOT** more details in order to be able to help you...

- client version?
- server version?
- code to reproduce the behavior you are seeing?

Thanks!

**Reply by llssyy1991** [reply]
Sorry, I think this might caused by some random stream generated the server side and that breaks the json format.

**Reply by darrachequesne** [reply]
I'm not sure to understand, is there something we could fix?

**Reply by llssyy1991** [reply]
Sorry, I don't think this is a socket.io issue any more, it should caused by other packages on python server side. Thanks for your help.
